openedx/features/discounts/docs/decisions/0001-determining-discount-location.rst

```diff
@@ -10,8 +10,24 @@ Context
 =======
 
 We are implementing platform-wide discounts that are determined using a combination of course and user. The
-data used in this determination lives in the lms. The discount will be messaged in the lms, and then autoapplied
-on the basket page of ecommerce.
+data used in this determination lives in the LMS. The discount will be messaged in the LMS, and then autoapplied
+on the basket page of Ecommerce. See the table below for information on how this new discount differs from our
+current discounting schemes.
+
++-------------------------------------+-------------------------------------+------------------------------------+ 
+| Discounts                           | Coupons                             | Enterprise Offers                  | 
```

Comment:
maybe note that discounts are also offers?

Reply:
Um...Do you mean that we are implementing the use of discount in Ecommerce by the Django-Oscar Offer framework?

Reply:
sure, that's what the word offers means in enterprise offers  
Potentially relevant because i'm sure we'll have offers that are not enterprise and not discounts in the future

Reply:
Coupons are also implemented as offers. I guess I just assumed that implementing it as an offer was standard.

Reply:
ah i think there was some confusion. i knew that coupons implement vouchers, but i guess vouchers themselves extend offers, so i guess the distinction i was suggesting doesn't exist